addons/sale_project/views/project_task_views.xml

```diff
@@ -28,6 +28,20 @@
         </field>
     </record>
 
+    <record id="view_task_kanban_inherit" model="ir.ui.view">
+        <field name="name">project.task.kanban.inherit</field>
+        <field name="model">project.task</field>
+        <field name="inherit_id" ref="project.view_task_kanban"/>
+        <field name="arch" type="xml">
+            <xpath expr="//templates" position="before">
+                <field name="allow_billable" invisible="1"/>
+            </xpath>
```

Comment:
Won't it be automatically added since it's used in the t-if?

Reply:
Good question, I don't know, I will check. 🙂

Reply:
Unfortunately, we have to define the field to use it inside the template, otherwise, the js code will crash because the field is not fetched.